docs/concepts/feathr-concepts-for-beginners.md

```diff
@@ -0,0 +1,115 @@
+
+Since those additional features are from different sources, we want to define an `Anchor` to process it further. Think `Anchor` as a `Feature View`, where it is a collection of features and their corresponding sources. Think `Feature` just as a column in your dataset but it contains some useful information that you want to use in your machine learning scenario.
+
+`Source` in Feathr just represents the source data that you will need to use to extract features from. It also comes with handy customizations that you can run almost arbitrary PySpark/SparkSQL code.
+
+That's why you will see something like below, where you define a Feathr Source (in this case it's an HDFS like source) and corresponding features, and then compose an `Anchor` that combines the `Features` and `Source`.
+
+```python
+batch_source = HdfsSource(name="nycTaxiBatchSourcePurview",
+                          path="wasbs://public@azurefeathrstorage.blob.core.windows.net/sample_data/feathr_delta_table",
+                          event_timestamp_column="lpep_dropoff_datetime",
+                          timestamp_format="yyyy-MM-dd HH:mm:ss")
+features = [
+    Feature(name="f_is_long_trip_distance",
+            feature_type=BOOLEAN,
+            transform="cast_float(trip_distance)>30"),
+    Feature(name="f_day_of_week",
+            feature_type=INT32,
+            transform="dayofweek(lpep_dropoff_datetime)"),
+]
+
+request_anchor = FeatureAnchor(name="request_features",
+                               source=batch_source,
+                               features=features)
+```
+
+## Motivation on `Derived Feature`
+
+That sounds all good, but what if we want to share a feature, and others want to build additional features on top of that feature? Thats's why there is a concept in Feathr called `derived feature`, which allows you to calculate features based on other features, with certain transformation support. In practice, people can build features on top of other features and have a "feature chain".
+
+One example for Derived Feautre is as below:
+
+```python
+f_trip_time_distance = DerivedFeature(name="f_trip_time_distance",
+                                      feature_type=FLOAT,
+                                      input_features=[
+                                          f_trip_distance, f_trip_time_duration],
+                                      transform="f_trip_distance * f_trip_time_duration")
+```
+
+## Why does Feathr need `Feature Join` and `Feature Query`?
```

Comment:
we are retiring the concept of feature join for public APIs. instead, we are using get_offline features. So i believe you don't need to talk about join.

Instead, you should say: how to access your features